hasktorch/src/Torch/Functional.hs

```diff
@@ -929,3 +943,19 @@ flattenAll
   -> Tensor -- ^ output
 flattenAll t =
   unsafePerformIO $ (cast3 ATen.flatten_tll) t (0 :: Int) (-1 :: Int)
+
+-- TODO: rename
+multinomial_tlb
+  :: Tensor
+  -> Int
+  -> Bool
+  -> IO Tensor
+multinomial_tlb t l b =
+  (cast3 ATen.multinomial_tlb) t l b
+
+-- TODO: rename
+bernoulli_t
+  :: Tensor
+  -> IO Tensor
+bernoulli_t =
+  cast1 ATen.bernoulli_t
```

Comment:
Basically, I am want to merge this pr.
I'm concerned about calling convention.

Reply:
yeah... I guess we can either figure out a plan for the names at #352 first or adjust them after we make a decision there, I'm open to any names here :woman_shrugging: